client/public/index.html

```diff
@@ -14,7 +14,9 @@
   <nav role="navigation" ng-controller="NavController" class="grey darken-3 z-depth-3">
     <div class="nav-wrapper container">
       <!-- Left Logo -->
-      <a id="logo-container" href="#" class="brand-logo white-text">WTF!</a>
+      <a  id="logo-container" href="/#/" class="brand-logo white-text">WTF!</a>
+      <a  ng-show="loggedIn" ng-class="{ dactive: isActive('/dashboard') }" id="logo-container" href="#/dashboard" class="brand-logo white-text">WTF!</a>
```

Comment:
RE: line #18, the ng-class="{ dactive: isActive('/dashboard') } causes the "wtf" logo to also highlight when it's clicked on if a user is logged-in. Removing it should do the trick.